phenotrex/ml/prediction.py

```diff
@@ -54,8 +62,9 @@ def predict(fasta_files=tuple(), genotype=None, classifier=None, min_proba=0.5,
         except TypeError:
             raise RuntimeError('This TrexClassifier is not capable of generating SHAP explanations.')
         sh = ShapHandler.from_clf(model)
-        sh.add_feature_data(sample_names=[x.identifier for x in gr],
-                            features=fs, shaps=sv, base_value=bv)
+        sh.add_feature_data(
+            sample_names=[x.identifier for x in gr], features=fs, shaps=sv, base_value=bv
+        )
```

Comment:
I currently do it manually when I feel like it - also since there are a few places where it looks like absolute crap (black does _not_ play well click decorators). Other people meditate, I reformat code. 🤷